pkg/kubectl/cmd/create/create_clusterrole.go

```diff
@@ -107,6 +113,13 @@ func (c *CreateClusterRoleOptions) Validate() error {
 		return fmt.Errorf("name must be specified")
 	}
 
+	if len(c.AggregationRule) > 0 {
+		if len(c.NonResourceURLs) > 0 || len(c.Verbs) > 0 || len(c.Resources) > 0 || len(c.ResourceNames) > 0 {
+			return fmt.Errorf("aggregation rule must be specified without neither nonResourceURLs, verbs, resources or resourceNames.")
```

Comment:
please change to:
`aggregation rule must be specified without nonResourceURLs, verbs, resources or resourceNames`
(notice no dot at the end)